server/docker/config/.env

```diff
@@ -1,5 +1,5 @@
 # Server
-HOST=0.0.0.0
+HOST=127.0.0.1
```

Comment:
So this will only listen locally.
I got confused a bit, but think this can cause some trouble on test env (AWS) (as that is using this env file).

I would revert this, if this change had no purpose on your side.

Reply:
Ok, checking back after reading activation email generation logic..
**Nevermind the comment above, I will add this env variable to AWS env.**

Reply:
It actually could behave differently on linux and windows if it was 0.0.0.0 or localhost. But 127.0.0.1 is more of a standard than those, every platform should understand it.